pkg/errors/generic.go

```diff
@@ -16,6 +16,7 @@ var (
 	ErrTagNotSupported     = "Tag unsupported: %s"
 	ErrTagAlreadyExists    = "Tag exists"
 	ErrTagDoesNotExist     = "Tag does not exist"
+	ErrTagNameInvalid      = "Tag name is invalid"
```

Comment:
They do not check for valid tagnames^^ https://github.com/go-playground/validator/blob/bd1113d5c1901c5205fc0a7af031a11268ff13ee/validator_instance.go#L154

What I would suppose and what I have seen somewhere else (do not remember where anymore), they called the tag name **field tag identifier**. This would make for more succinct error names.

```go
ErrFieldTagIdentifierInvalid = "Field tag identifier invalid"
```

Apart from this, one could make it clear for the user of the package, what went wrong, by copying the approach of `ErrTagNotSupported` and use `ErrFieldTagIdentifierInvalid = "Field tag identifier invalid: %s"` in conjunction with `fmt.Errorf(ErrFieldTagIdentifierInvalid, "<bad-tag-identifier>")`

Reply:
ErrFieldTagIdentifierInvalid  LGTM, can u help to update

Reply:
Done.